docs/workflows/common-workflow-steps/appshield-scanner.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Appshield Scanner for Android/iOS
+description: Appshield Scanner performs dynamic runtime scans/analysis and AI supported static (file-based) inspections for mobile app files (APK, AAB, IPA) to detect existing security mechanisms and indicates whether an app is secure or not.
+tags: [android, ios, mobile, security, scan, appshield-scanner]
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
```

Comment:
There is a ready-to-use component for Sensitive vars. Please update it as follows:
```suggestion
import SensitiveVariablesDanger from '@site/docs/\_sensitive-variables-danger.mdx';

```